README.rst

```diff
@@ -2,8 +2,6 @@
 PyModbus - A Python Modbus Stack
 ================================
 
-.. image:: https://travis-ci.org/riptideio/pymodbus.svg?branch=master
```

Comment:
We should add github badge instead, now that the CI is running in github itself.
https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge

Reply:
This gives the badge for `dev` branch
https://github.com/riptideio/pymodbus/actions/workflows/ci.yml/badge.svg?branch=dev

Reply:
Added.